specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/CommonDefinitions.json

```diff
@@ -96,6 +86,29 @@
         }
       }
     },
+    "Container": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/InitContainer"
+        },
+        {
```

Comment:
>  { [](http://example.com/codeflow?start=6&length=3)

This is causing the error I think. The the correct example here [https://github.com/Azure/azure-rest-api-specs/blob/f842bc69f4c215ad60ffae63ef517c201772f8b2/specification/app/resource-manager/Microsoft.App/stable/2022-03-01/ContainerApps.json#L463](https://github.com/Azure/azure-rest-api-specs/blob/f842bc69f4c215ad60ffae63ef517c201772f8b2/specification/app/resource-manager/Microsoft.App/stable/2022-03-01/ContainerApps.json#L463)